ci/nightly/pipeline.template.yml

```diff
@@ -1663,6 +1663,16 @@ steps:
       - ./ci/plugins/mzcompose:
           composition: backup-restore-postgres
 
+  - id: 0dt
```

Comment:
The rehydration tests with real large data are quite slow, 90 min total for the test currently. I find them important enough to still do, but moved the test to nightly for now

Reply:
Do you know where most of the time goes? Is it in generating enough data to slow the rehydration time down?

Reply:
Yes, exactly.